qutebrowser/config/configdata.yml

```diff
@@ -36,6 +36,12 @@ history_gap_interval:
 ignore_case:
   renamed: search.ignore_case
 
+crash_count:
+  type: Int
+  default: 3
+  desc: >-
+    Maximum times the renderer process can crash before we stop restarting it, use -1 to disable restarting.
```

Comment:
Could you [wrap this line below 80 characters](https://github.com/jcjordyn130/qutebrowser/commit/99ad1547bc27bd99ef588fed9f900f19f00f41a1#diff-723dbd7f6c5e462644bbea6d72ae0adf)? :wink:
Also, the 'use -1...' part is written as a new sentence for other options, I think.
How about 'Maximum number restarts after renderer process crashes. Use -1 to disable restarting.'?

Reply:
That sounds better actually.